### ISSUE TITLE
Provides graphical monitoring

### ISSUE THREAD

**Issue by wxpid1** [kingToolbox/WindTerm#2070]
Icon monitoring is still not intuitive enough
![image](https://github.com/kingToolbox/WindTerm/assets/45633931/a3e3623d-88e2-4128-8f41-f3632f94f51c)
Graphical monitoring will be more intuitive, one can see the state of the system, but also to the software will be a higher level
![1705203855411](https://github.com/kingToolbox/WindTerm/assets/45633931/5ab25aa2-9a7c-4dc8-9261-0cd2dd77edff)

**Comment by wxpid1** [comment]
Click on the corresponding category to see the corresponding details
![1705204812435](https://github.com/kingToolbox/WindTerm/assets/45633931/69e54d1f-17a2-4c2c-9a1f-a50043596515)

**Comment by kingToolbox** [comment]
Yes, excellent data visualization. This feature has already been planned, and I will implement it in version `3.x`. Please stay tuned, thank you.
